### PR TITLE
Nukie Lobby Art

### PR DESCRIPTION
**Pull request by cheki-ss14**
## About the PR

This PR is for an addition to the pool of lobby art. I have added a piece that I've created and believe it would make a great addition to the lobby art in game.

## Why / Balance

N/A - Doesn't affect anything in game.

## Technical details

N/A 

## Media

IMG_0127

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

:cl:
- add: Added a new splash screen lobby art!

### CODE REVIEW

**Comment by cheki-ss14** [comment]
Please feel free to reach out on discord @cheeki or on here if I have done anything incorrectly. This is my first time ever doing anything like this.

**Comment by pigeonpeas** [comment]
extremely based, sick art

**Comment by luizwritescode** [comment]
goofy linearts are cool but would love to see more lobby art that plays with realism like this one does.

File: Resources/Textures/LobbyScreens/attributions.yml [hunk 1]
```diff
@@ -56,4 +56,9 @@
 - files: ["reclaimer-nuke.webp"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "GetOutMarutak,aka.Snicket on Discord/github/Cara/Ko-fi"
-  source: "https://github.com/space-wizards/space-station-14"
\ No newline at end of file
+  source: "https://github.com/space-wizards/space-station-14"
+
+- files: ["nukie_commander.webp]
```
Comment (Cojoke-dot) [inline Resources/Textures/LobbyScreens/attributions.yml:61]: forgot the quotation marks at the end of this
["nukie_commander.webp"]

**Review by VasilisThePikachu — APPROVED** [review]

**Comment by ShadowCommander** [comment]
Could you add more to the image? Maybe stuff in the background like posters that were suggested in Discord? It'd be nice to have more to look at during the two minute lobby time than just the nuke op.

**Comment by cheki-ss14** [comment]
> Could you add more to the image? Maybe stuff in the background like posters that were suggested in Discord? It'd be nice to have more to look at during the two minute lobby time than just the nuke op.

I had attempted to add some of the suggestions that were in the Discord such as posters or glowing text behind the nukie and just wasn't fully satisfied with the way it turned out. However if that's what you believe is best then I shall spend more time on the piece and update it. Thanks for the suggestion!

**Comment by UbaserB** [comment]
You could probably add posters or a window, if you take the stance that the background is a wall.

**Comment by slarticodefast** [comment]
Are you coming back to this?

**Pull request closed without merging**